src/test/java/unicash/model/category/UniqueCategoryListTest.java

```diff
@@ -73,7 +73,7 @@ public void add_maxCategory_throwsMaxCategoryException() {
         for (int i = 0; i < UniqueCategoryList.MAX_CATEGORIES; i++) {
             uniqueCategoryList.add(new Category("Test" + i));
         }
-        assertThrows(MaxCategoryException.class, () -> uniqueCategoryList.add(new Category("test4")));
+        assertThrows(MaxCategoryException.class, () -> uniqueCategoryList.add(new Category("test5")));
```

Comment:
this line was modified because the case-insensitivity of the categories were causing `test4` to be flagged as a duplicate